### ISSUE TITLE
Inconsistent horizontal scroll behaviour across platforms

### ISSUE THREAD

**Issue by falkTX** [lv2/pugl#113]
During my testing with a Linux laptop with a trackpad, I noticed that horizontal scroll behaves the inverse between Linux and macOS.
I do not have direct access to that machine right now, but want to report the issue for posterity.
I had the issue in https://github.com/DISTRHO/Cardinal/issues/582 where I noticed on a Linux machine that it was inverted, so fixed that by inverting on the plugin side, but then that made it wrong on macOS.

Basically the horizontal scroll on either macOS or Linux is inverted between each other.
I was unable to try this on a Windows machine, so do not know what is the state there.

**Comment by drobilla** [comment]
Seems it was reversed on MacOS compared to X11 and Windows.  The speeds with trackpads are wildly inconsistent, it looks like some explicit support for high resolution is needed, but direction fixed in 4d9059e anyway.  I'm not able to test on MacOS with a mouse with a detented wheel.
